src/frontend/src/components/public/PIAFormTabs/review/index.tsx

```diff
@@ -167,38 +179,37 @@ const PIAReview = () => {
-                        <div className="d-block mt-3">{role}</div>
-                        <div className="d-block">
-                          <button
-                            className="bcgovbtn bcgovbtn__tertiary bold min-gap delete__btn mt-3 p-3"
-                            onClick={() => {
-                              reviewForm.programArea.selectedRoles?.splice(
-                                index,
-                                1,
-                              );
-                              stateChangeHandler(
-                                reviewForm.programArea.selectedRoles,
-                                'programArea.selectedRoles',
-                              );
-                              piaStateChangeHandler(
-                                {
-                                  programArea: {
-                                    selectedRoles:
-                                      reviewForm.programArea.selectedRoles,
-                                  },
-                                },
-                                'review',
-                                true,
-                              );
-                            }}
-                          >
-                            <FontAwesomeIcon
-                              className="ms-1"
-                              icon={faTrash}
-                              size="lg"
-                            />
-                          </button>
+                        <div
+                          className={`d-block mt-3 ${
+                            pia.status === PiaStatuses.FINAL_REVIEW && 'mb-2'
+                          }`}
+                        >
+                          {role}
                         </div>
+                        {pia.status !== PiaStatuses.FINAL_REVIEW && (
+                          <div className="d-block">
+                            <button
+                              className="bcgovbtn bcgovbtn__secondary--negative mt-3 ms-3 bold min-gap delete__btn p-3"
+                              onClick={() => {
+                                reviewForm.programArea.selectedRoles?.splice(
+                                  index,
+                                  1,
+                                );
+                                stateChangeHandler(
+                                  reviewForm.programArea.selectedRoles,
+                                  'programArea.selectedRoles',
+                                );
+                                piaStateChangeHandler(reviewForm, 'review');
+                              }}
+                            >
+                              <FontAwesomeIcon
+                                className="ms-1"
+                                icon={faTrash}
+                                size="lg"
+                              />
+                            </button>
+                          </div>
+                        )}
```

Comment:
Also, I am assuming you will have a follow up PR for program area "reviews" in the FINAL status. Is that correct?

Reply:
That's correct. This one is for missing UI related to the _delegated_ flow which is what our sprint goal is focussed on.

Reply:
Will _delegated flow_ not have any program area reviews?

Reply:
<img width="296" alt="image" src="https://github.com/bcgov/cirmo-dpia/assets/87394256/4729c9e3-58f4-47fa-8a79-90ce651340c1">

<img width="737" alt="image" src="https://github.com/bcgov/cirmo-dpia/assets/87394256/4a8a2933-4ba0-4ce1-9ec9-8e0b76e81e93">

Reply:
it should have program area for delegate PIA as well this is my understadning